### DISCUSSION TITLE
Advice on image processing and SimpleITK channel arithmetic

### DISCUSSION THREAD

**Discussion by LJY1207** [IBEXImagingCommunity/ibex_imaging_knowledge_base#211] [Q&A]
Hello, 

I'm very new to image processing/analysis and am currently trying to process images to make them publication-ready. I've had a lot of trouble with RBC autofluorescence and just autofluorescence/non-specific signal in general. I've attached an example of an image I'm working with, where the transcription factor SOX9 (red) should have nuclear localization in the biliary epithelium and peribiliary glands of the cystic duct but also seems to appears to stain non-specifically. 

What is the best way or standard practice to approach something like this? Is the nonspecific signal something I should remove using SimpleITK channel arithmetic (such as subtracting anything that isn't nuclear), or is it fine to leave in the images, but I then only analyze SOX9 positive signal that is also positive for Hoechst? Or is this something that should have been addressed way back in the staining/antibody phase? (In that case I am out of luck, as I am no longer with the lab and am finishing making figures virtually...)

Cheers and thank you in advance for any input!

Blue - Hoechst (405)
Green - CK19 (AF647)
Red - SOX9 (AF488)
Cyan - SMA (AF750)

![20240403_Michigan_CBD_IBEX_Panel1_Panel2_overlay_SimpleITK_2024-08-11T11-56-09 765](https://github.com/user-attachments/assets/e5088a92-e79d-4a55-be7c-1e9b329add07)

![20240403_Michigan_CBD_IBEX_Panel1_Panel2_overlay_SimpleITK_2024-08-11T12-03-46 740](https://github.com/user-attachments/assets/c58e0dc0-419d-4d39-b790-68234650d636)

Best,
Yuri

**Comment by radtkea** [comment]
Hi Yuri,

Thanks very much for your question! It's hard to answer without looking at the images. Let's schedule a time to chat. We will work on a solution and share what we find. Fear not!

You can email me at andrea.radtke "at" nih.gov.

Best wishes,
A
